### ISSUE TITLE
Access to Pytorch Lightning auto_lr_find  functionality seems broken [BUG]

### ISSUE THREAD

**Issue by solalatus** [unit8co/darts#1403]
**Describe the bug**

When using Pytorch based models, Lightning's auto_lr_find functionality does not seem to be accessible.

**To Reproduce**
When initializing a Pytorch based model, eg.:

`model_nhits = NHiTSModel(...
pl_trainer_kwargs = {"auto_lr_find":True,...} ...)`

The LR tuning feature described [here](https://pytorch-lightning.readthedocs.io/en/stable/advanced/training_tricks.html#learning-rate-finder) does not seem to get executed.

By manual tries to execute the tuning procedure by:

`lr_finder = model_nhits.model.trainer.tuner.lr_find(model_nhits.model, train_dataloaders=[model_nhits.model.trainer.train_dataloader])` 

one gets the error:
`
MisconfigurationException:
                If learning rate/lr in 'model' or 'model.hparams' isn't overridden when 'auto_lr_find=True',...`

**Expected behavior**
- auto_lr_find to have effect
- manual access to .tune() and it's plot output, which is quite useful for say setting up max learning rates in case of 1cycle policy.

**System (please complete the following information):**
 - Python version: 3.7.15
 - darts version 0.22.0

**Additional context**
It is entirely possible, that there is an obvious way to get these things to work, so please point me to any if! I found sadly none.

**Comment by solalatus** [comment]
In a really not intuitive way, it happens to work.

For further reference:

`lr_finder = model_nhits.model.trainer.tuner.lr_find(model_nhits.model, train_dataloaders=model_nhits.model.trainer._data_connector._train_dataloader_source.instance)`
